src/conversions/std/array.rs

```diff
@@ -1,308 +1,118 @@
+use crate::conversion::{AsPyPointer, IntoPyPointer};
```

Comment:
The diff in this file looks messy due to the indentation changes but I did not touch a single line of code. I just merged the inner modules in the outer one, i.e. I removed and moved code.